### PR TITLE
Refine build and install tutorial doc for paddle inference

### PR DESCRIPTION
**Pull request by cryoco**
Refine build and install tutorial doc for paddle inference.
-- updated the download source of paddle inference lib package to 1.6.
-- added download source of inference lib with TensorRT(cuda8 -> TRT4, cuda9&cuda10 -> TRT5).
10 162 69 147_8889_documentation_docs_zh_develop_advanced_usage_deploy_inference_build_and_install_lib_cn html

### CODE REVIEW

File: doc/fluid/advanced_usage/deploy/inference/build_and_install_lib_cn.rst [hunk 1]
```diff
@@ -28,7 +30,7 @@
 ============================  =========
 CMAKE_BUILD_TYPE              Release
 FLUID_INFERENCE_INSTALL_DIR   安装路径
-WITH_PYTHON                   OFF（推荐）
```
Comment (luotao1) [inline doc/fluid/advanced_usage/deploy/inference/build_and_install_lib_cn.rst:31]: 这行为什么要改呢？C++库不需要python

File: doc/fluid/advanced_usage/deploy/inference/build_and_install_lib_cn.rst [hunk 2]
```diff
@@ -6,18 +6,20 @@
...
+    "ubuntu 14.04","cuda9.0_cudnn7_avx_mkl", 4.8, "`fluid_inference.tgz <https://paddle-inference-lib.bj.bcebos.com/1.6.0-gpu-cuda9-cudnn7-avx-mkl/fluid_inference.tgz>`_", "`fluid_inference.tgz <https://paddle-inference-lib.bj.bcebos.com/latest-gpu-cuda9-cudnn7-avx-mkl/fluid_inference.tgz>`_"
+    "ubuntu 14.04","cuda10.0_cudnn7_avx_mkl", 4.8, "`fluid_inference.tgz <https://paddle-inference-lib.bj.bcebos.com/1.6.0-gpu-cuda10-cudnn7-avx-mkl/fluid_inference.tgz>`_", "`fluid_inference.tgz <https://paddle-inference-lib.bj.bcebos.com/latest-gpu-cuda10-cudnn7-avx-mkl/fluid_inference.tgz>`_"
+    "ubuntu 14.04","cuda8.0_cudnn7_avx_mkl_trt4", 4.8, "`fluid_inference.tgz <https://paddle-inference-lib.bj.bcebos.com/1.6.0-gpu-cuda8-cudnn7-avx-mkl-trt4/fluid_inference.tgz>`_", 
+    "ubuntu 14.04","cuda9.0_cudnn7_avx_mkl_trt5", 4.8, "`fluid_inference.tgz <https://paddle-inference-lib.bj.bcebos.com/1.6.0-gpu-cuda9-cudnn7-avx-mkl-trt5/fluid_inference.tgz>`_", 
+    "ubuntu 14.04","cuda10.0_cudnn7_avx_mkl_trt5", 4.8, "`fluid_inference.tgz <https://paddle-inference-lib.bj.bcebos.com/1.6.0-gpu-cuda10-cudnn7-avx-mkl-trt5/fluid_inference.tgz>`_",
```
Comment (luotao1) [inline doc/fluid/advanced_usage/deploy/inference/build_and_install_lib_cn.rst]: gcc版本和ubuntu版本不需要单独加一列，因为都是4.8，只要说明一下就可以了。

**Comment by cryoco** [comment]
@luotao1 
Done, thanks.
10 162 69 147_8889_documentation_docs_zh_develop_advanced_usage_deploy_inference_build_and_install_lib_cn html

**Review by luotao1 — APPROVED** [review]

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**